docs/site/src/app/pages/limit-policy/limit-policy.component.scss

```diff
@@ -0,0 +1,72 @@
+:host {
```

Comment:
Avoid styling components via classes instead you should tightly couple them to their styles by specifying more strict selectors.